lib/puma/cluster.rb

```diff
@@ -5,6 +5,7 @@
 require 'puma/plugin'
 
 require 'time'
+require 'json'
```

Comment:
Currently we only require `json` for the status app, which isn't that big of a deal. This would definitely require `json` for most people running Puma.

Reply:
For example, this would, I believe, introduce a new native extension during startup, which I believe affects other open PRs.

Reply:
I thought requiring `json` would be straightforward as part of the [standard library](https://ruby-doc.org/stdlib/libdoc/json/rdoc/JSON.html), but I did find it unusual that it was being intentionally avoided. If loading `json` does introduce some known stability/performance issue on any of the supported platforms (I'm less familiar with non-linux/MRI platforms) it makes sense to not accept this change.

Reply:
I checked with Evan and there was no special reason. This is good to 🚢